### ISSUE TITLE
Language code plugin

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#25808]
The plugin has a link to w3.org

![image](https://user-images.githubusercontent.com/1296369/62825232-5531cf80-bba0-11e9-87ca-88ce0e4c7bfe.png)

I have no idea why that link was deemed to be relevant but it certainly is not any more especially as the link https://www.w3.org/TR/xhtml1/#docconf is a superseded recommendation it is definitely wrong

**Comment by infograf768** [comment]
That was useful in 2.5 when we were using `xmlns` and was never modified.

Example in 2.5
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="fr-fr" lang="fr-fr" dir="ltr" >
<head>
```

Can be modified indeed.

**Comment by brianteeman** [comment]
Can the link be removed? Or should we search for another link? I think it can be removed.

**Comment by infograf768** [comment]
it can be removed if we are sure that all templates around are not using any other DOCTYPE than the default protostar (or  Cassiopea in 4.0)  `<!DOCTYPE html>`
 or replaced by `https://www.w3.org/International/questions/qa-html-language-declarations`

**Comment by ghost** [comment]
Closed as having Pull Request #25811
